### PR TITLE
fix/styling in user admin

### PR DESCRIPTION
**Pull request by Her0elt**
fix/styling in user admin
Lint, test and build
The styling in user admin was a bit of center this should fix it

Comments/issues/screenshots:

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] PR name is descriptive
- [ ] The PR includes the link to the relevant Trello task
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

- [ ] `yarn build` was run locally without errors
- [ ] `yarn test` has passed locally and any fixes were made for failures
- [x] `yarn format` was run locally and fixes were made for new errors

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: src/containers/UserAdmin/tabs/Members.js [hunk 1]
```diff
@@ -238,22 +239,20 @@ const Members = (props) => {
...
+                <Typography className={classes.title} variant='subtitle1'>
+                  Studie:
+                </Typography>
+                <Typography className={classes.title} variant='subtitle1'>
+                  Klasse:
+                </Typography>
+              </Grid>
             </Hidden>
             <Pageination fullWidth nextPage={getNextPage} page={nextPage}>
               {memberList &&
```
Comment (olros) [inline src/containers/UserAdmin/tabs/Members.js:258]: Ikke egt en del av denne pr'en, men når vi printer ut medlemmene så trenger vi ikke en div rundt hver av dem.
Dette:
```
<div key={index}>
  <PersonListItem data={value} handleMembers={handleMembers} isMember={props.isMember} />
</div>
```

Kan byttes ut med dette:
```
<PersonListItem data={value} handleMembers={handleMembers} isMember={props.isMember} key={index} />
```

Fikser du?

File: src/containers/UserAdmin/tabs/Members.js [hunk 2]
```diff
@@ -41,7 +40,8 @@ const styles = (theme) => ({
     gridGap: '10',
     width: '100%',
     textAlign: 'left',
-    gridTemplateColumns: '2fr 1fr 1fr 1fr',
+    padding: 5,
+    gridTemplateColumns: '5fr 3fr 3fr 2fr 2fr',
```
Comment (olros) [inline src/containers/UserAdmin/tabs/Members.js:44]: Denne bør være `3fr 2fr 2fr 2fr 48px` siden medlemmsradene er `3fr 2fr 2fr 2fr` med et ikon til høyre som er `48px`

**Review by olros — CHANGES_REQUESTED** [review]

File: src/containers/UserAdmin/tabs/Members.js (hunk 1 above)
Comment (Her0elt) [reply src/containers/UserAdmin/tabs/Members.js:258]: yes

**Review by olros — APPROVED** [review]
Niice 💯

**Pull request merged**